### ISSUE TITLE
Sheet Moderation

### ISSUE THREAD

**Issue by Charanraja123** [Gr770/CK3-Community-Title-Project#48]
Okay. This is kind of a nuanced topic, and I am sharing my personal opinions on it. 

The sheet needs to be cleaned up/pruned. There are lot of unnecessary entries. Definitions for rows are being misinterpreted (prince_rulers) and some duplicates too. Considering we have a sizable amount of entries and more on the way, this is high time we get it into a more organized state

Since we have already established some structure with the Mod side, i feel that should be reflected accurately in the sheet, since it is the best model we have. 

For Example, There is no special column for an independent tribal ruler (he falls under principality). Why is there republic kingdoms?. I think that is so niche of a topic that it does not warrant a row for itself.

Also, it would be nice if someone else could take share the work with @Viridianus in organizing the sheet. I would like to know if you guys (especially @Viridianus) share this feeling of having someone else help him in maintaining the sheet might help.

These are all just opinions that I felt I could share, I apologize if I have offended anyone.

**Comment by Viridianus** [comment]
Organizing the sheet is good - even though I don't quite see what you mean by unnecessary entries (except for republic kingdoms: these are not niche as this is how Italian influential republics and Hansa are treated, AFAIK).

Coding side is restricted by things which are coded in different places while logically belong to the same.

As for principalities for tribal rulers - that's not how the sheet is supposed to work. Principality (and prince(ss)-ruler) is feudalclan_independent, tribality takes priority over that (at least in theory; I will not be surprised if it is coded differently). On republic kingdoms see above.

This will sound unnecessarily harsh, but right now I feel like any "helper" with either formatting or EtB first column will only cause me to wanna tear their arms out of their body and reinsert them (note that while I use Necro's localization extraction and am grateful to him/her for it, it wouldn't even cross my mind to suggest s/he should insert them in the first column personally). But there are many other tasks in maintaining the sheet, including reviewing (which @Gr770 currently does), expanding when needed (which nobody usually does; for instance, we probably need a row for culture groups, as I posted in the forum) and rearrangement (ditto, the only large rearrangement was at September 1).

**Comment by Charanraja123** [comment]
Let me correct myself here. 
By unnecessary entries, I meant to cite republic kingdoms. I still think something special to Italy should go into conditional titles, but I'm happy to defer to your judgement on this.
The tribal point is interesting since actually, I did do it for feudal clan only afaik. but i'm sure I saw some principalities with tribal governments, which now we should remove(right?). Also this means the "independent" tag doesn't mean anything for tribal governments since we do not have anything specific for that.
You already raised the Issue of duplication, which is something we both agree on.
I definitely am onboard for reorganization. At least getting in ascending/descending order of Tier would be really nice. 

P.S. it is he/his/him, saving you some time in the pronouns friend xD

**Comment by Viridianus** [comment]
;)
I don't think it should be special to Italy. Hansa is an obvious example outside, but also - historically Novgorod republic reached kingdom-level at some point, controlling... well, what's k_novgorod AND k_bjarmaland on the map. They had a knyaz', of course, even if in a limited function, so Novgorod specifically could go with Velikoje knyazhestvo, but this is a general example.
Oh, there's an interesting issue: it is quite possible that _the actual titles_ in some cultures are the same for tribes and for feudalclan_principalities. I only meant logic of the sheet.

**Comment by Charanraja123** [comment]
Like the Sultan, Emir and Emirate rows being used for non-Islamic faiths, Shouldn't we add similar Christian rulers for non-Christian faiths? Also what does this mean since I see sultan under English, but was there any historical significance behind it? is it supposed to be like a translation?

**Comment by Viridianus** [comment]
I assume that @Gr770 believes (or believes Pdx to believe) that, since Arabic Islamic rulers distinguished Malik and Sultan, every Islamic ruler should distinguish King and Sultan. The rows, of course, should not be used for non-Islamic faiths; this pertains to the general question of defaults.

**Comment by Gr770** [comment]
Sultan signifies a faith-based submission to a higher power (Historically the caliph). Religous submission is an important part of the Islamic faith.

Emirs is another story though.

**Comment by Viridianus** [comment]
So, @Gr770, you were lured and came here ;)
1)Why do we have Emir and Sheikh rows?)
2)Do we officially accept the measure on marking defaults as such instead of repeating the title?
3)What do you have to say on the rest?

**Comment by Charanraja123** [comment]
I do understand the role and difference between sultan/malik. My questions mainly pertained to why is it included as row in general and is it expected to have English entries into it? 
The submission to the caliph is something more into the religious condition which we haven't discussed yet. Including arabic christians, islamic indians and such. There is a discussion required in how to represent these on the sheets so that we can look at it in a more holistic manner.

**Comment by Gr770** [comment]
1.) I know sheikh is more of religious title, but I'm not opposed to removing them. Sultan should probably stay as that was in the game as is.
2.) I've tried labeling 'defaults to' on the sheet, but not everyone sees it. I'm planning on going through the sheet and expanding and co tracting it after work. I can leave you for EtB
3.) Republics should stay for future proofing, I don't really have a good solution for labeling tribal independents right now.

**Comment by Charanraja123** [comment]
I did not suggest removing Sultan/Malik, Just make it more clear that that and subsequent muslim ruler row are to entered for cultures which were multi-religious and do not have Islam cultures as their default. Iberian or Indo-Aryan cultures would be the perfect example. Having sultan entries for English just feels wrong.
Maybe give it a row gap with a description of something 
In a similar fashion we could also have Christian based Emperor/King/Queen etc titles which can be used for some non-European Christians i.e, where the default king/queen does not have a christian Annotation like the existing Coptic Egyptian. Again these dont have to be filled up unless it applies to that particular culture, like the sultan/malik/sheik stuff.
Hopefully you guys can understand me.

**Comment by Charanraja123** [comment]
1) Yes, Sultan will remain, but please make sure people dont misinterpret it as a translation. I'm sorry if i'm repeating my point, but I just think there is a chance for miscommunication. Since we are talking about sultan, The sultan/malik difference is something that we can address after having flags included by Paradox. I did dirty hack it for the current arabian version, but I do not want to do that for everything. (Said this since we were on this topic).
2) I vote forleaving Intact, since the game by itself defaults. It helps to make sure you have modded the only the required things
3) Hard Agree. Independent Tribal is like a Tautology lol. I do not want a distinction based on independence. If you all agree, I'll add removing Tribal from Principalities into the Fixes issue

**Comment by Charanraja123** [comment]
4. Sorry for bringing this up late. But can we include a Petty Kingdom row? since a lot cultures seemed to want it.

**Comment by palfourti** [comment]
Is "prince ruler" not supposed to represent petty kingdoms"? Maybe I'm one of those people who's misunderstood that row but I assumed it was meant to be for things like letting independent duke/count rulers still refer to themselves as kings if it made sense for their culture (Norse, Anglo-Saxon, etc.)

**Comment by Charanraja123** [comment]
I specifically used it only for counts/dukes since it didn't say anything about kingdom lol. Another one to clean up i guess

**Comment by Gr770** [comment]
Yeah, I'll label principality as principality/petty king from now on. They are functionally identical but some cultures (irish) wouldn't really have principalities.

**Comment by Gr770** [comment]
Also knights need to be changed a bit.

**Comment by Charanraja123** [comment]
How do you mean?

**Comment by Gr770** [comment]
There is
Knight
Knight_plural
Knight_pural_no_tooltip

**Comment by Charanraja123** [comment]
> 
> 
> There is
> Knight
> Knight_plural
> Knight_pural_no_tooltip
> 
> In the files by default.

So knight f ?. I assumed that to be the case. Since we are only changing the word knight and not giving actual titles.

**Comment by Gr770** [comment]
Yes

**Comment by Viridianus** [comment]
What's the difference between knight_plural and knight_plural_no_tooltip in localization files now?
1)I'll delete the rows today (or tomorrow - but probably today).
2)I mean, this is a source of miscommunication. I suggested putting a special symbol like ~ or '= to affirm defaulting.
3)Sounds agreeable (if not already done).

**Comment by Charanraja123** [comment]
So basically there are different versions of Knight localisations used in events and stuff. No tooltip is for places that they dont want to show a tooptip, whereas tooltip is somewhere they want to show a tooltip.  In a similar fashion, there is singluar/plural and Title case /lowercase

**Comment by Viridianus** [comment]
I mean, are they ever localized differently, or is it just a sign that we will have to duplicate it each time?

**Comment by Charanraja123** [comment]
I don't quite understand your question. But if you are asking whether we have write "knight" in eight different version (left out adjective and plural possessive) for the localisation, then yes we have to do them.

**Comment by palfourti** [comment]
Since we're discussing pruning unnecessary entries from the title sheet I wanted to ask, do we really need a Grand Duchy row? Maybe @Gr770 or someone else can correct me but from what I've seen the only instant of "Grand Duke" in the game is as a special cultural localization for independent Lithuanian king-level rulers, and that there's no special "grand duchy" tier or government type as it were. If that's the case there's really no reason for that row to exist since every culture will already have its own localization for king titles.

**Comment by Viridianus** [comment]
AFAIK, Grand Duke was supposed to be a title for an independent holder of multiple duchies who is not a king (so somehow greater than mere Prince ruler); however, @Gr770  hasn't even found a way to implement them. I would go against deleting the rows for now, but it may turn out that we don't really need them.

**Comment by Gr770** [comment]
Move Marches, Palatinates, and Grand duchies to a possible sheet?

**Comment by palfourti** [comment]
Ah, I wasn't aware, I assumed it was to fill something already in the game.

**Comment by Viridianus** [comment]
No, please don't multiply sheets (among other things, it makes working with requested cultures way harder). Maybe a special section in the end of relevant tables?

**Comment by Viridianus** [comment]
Also, I request protection for columns of Bosnian and Buryat on Main Titles sheet (as well as first column on EtB sheet). Someone just tried to insert a Croatian term for Bosnians.

**Comment by Gr770** [comment]
I cant do it from my phone, but I'll check updates from my time till when I can lock it and reverse them.

**Comment by Viridianus** [comment]
No need for reversing, THAT I took care of) What do y'all think of a special section for "possible titles" on MT and RC sheets?

**Comment by Gr770** [comment]
I put a warning up for editing the titles. I don't feel it needs a total lockdown, I don't want to completely discourage someone from posting a source.

**Comment by Gr770** [comment]
In fact I just went ahead and placed a warning for all completed (mostly green) cultures.

**Comment by Charanraja123** [comment]
Everything seems amazing guys. Overnight you guys build Rome xP.
Some changes, The titles "Council and Court","Republics" are not really visible. If you could make the same changes to them as "Possible Future Titles" would be nice.

**Comment by Charanraja123** [comment]
Also someone should review the existing Sultan entries.

**Comment by Viridianus** [comment]
I will raise the issue of defaults here until it is solved :D I really want to use ~ there, but then do we color the tildas same as usual titles?
And, related, East Slavic female barons should actually distinguish regnants and consorts - that was my bad, and I edited the sheet now accordingly.

**Comment by Charanraja123** [comment]
Color the tildas the same way as any other entry i suppose. If someone says this shouldn;t default, they they color it yellow. If we modded it in, we color it green.
On a related note why does the rules say tribal titles overrides prince rulers? aren't they entirely different government types? I'm okay with them defaulting to count/duke. But I cant understand the override thing

Also. English sultan. Someone. Please. xP

**Comment by Viridianus** [comment]
Included the rule on tildas in RaG. Commented on Sultan in the sheet.

Also, I petition for deleting third row in EtB: it duplicates information from Main Titles and isn't updated accordingly. Same pertains to Religious Titles.

**Comment by Viridianus** [comment]
Do you remember me saying that someone else's formatting will make me wanna tear their arms out? Religious Titles was a prime example of this.

**Comment by Gr770** [comment]
1. For deleting the third row, are these named the same as the head of faith or are they the names of the church?

2. I mostly added all the faiths in the files. Feel free to move them around.

**Comment by Viridianus** [comment]
The third row is cultural names of cultures (Erennach for Irish and so on). It's enough to have the row in MT (and RC).

**Comment by Viridianus** [comment]
Turns out that because of lock on EtB first column you can no longer hide/unhide, so @Gr770 - please unhide all the lines.

**Comment by Gr770** [comment]
I set it to warning like the others

**Comment by Viridianus** [comment]
Still can't unhide (just like I can't now hide lines in MT).

**Comment by Gr770** [comment]
That is a little odd. I just went ahead and removed the protections.

**Comment by Viridianus** [comment]
Thx! That also led me to recheck Pdx's localization of no-hierarchy titles and update them (as well as get rid of that 3rd row). You can re-protect the first column now (hiding those few titles is too minuscule given the twelve thousand titles anyway).

Also, do I really have to say that any rearrangements of MT are to be repeated in RC?

**Comment by Charanraja123** [comment]
So.. can we bring this up to 50 comments. It sounds like a good number and I would be proud of creating this Thread xD

**Comment by Viridianus** [comment]
Unfortunately, it doesn't even seem to be difficult. There are multiple problems - for instance, many important edits are commented and remain unimplemented for weeks despite having a good proof while some less-proven edits belonging to more self-assured people get in. Someone should go through comments regularly - and preferably more than one (I can't do the whole sheet moderation alone, even just for MT it's thousands of cells, not to mention it begins to quickly develop into me making judgment calls alone because no one else cares).

**Comment by Charanraja123** [comment]
So you need someone to help you moderate the sheet. Honestly post it in the forum. I'm sure there are people eager to contribute to this. Also there were a "few" people who wanted to come in, Do you have any idea what happened to that?

**Comment by Gr770** [comment]
To be honest, I'm not doing too much here actually.

My problem is that I work for the national guard and country is still coming to terms with the fact that cops aren't supposed to kill people. I'm just working longer days for now at the maintenance shop.

I didnt want to bring it up on the forums, but I really appreciate the work you have all done.

**Comment by Viridianus** [comment]
You still find some time for us ;)

Btw, what do you think of splitting count-level and duke-level Petty Kings in the master sheet (each defaulting to the corresponding non-independent title)? I finally understood what bothers me that much about the principalities :D

**Comment by Charanraja123** [comment]
> 
> 
> To be honest, I'm not doing too much here actually.
> 
> My problem is that I work for the national guard and country is still coming to terms with the fact that cops aren't supposed to kill people. I'm just working longer days for now at the maintenance shop.
> 
> I didnt want to bring it up on the forums, but I really appreciate the work you have all done.

Hey, From what I see, you do all the merges, have the most commits and additions and gave me a platform to have fun breaking a game. I'm satisfied. Also its fun to watch @Viridianus threatening to tear off his own arms. Still waiting for it xP

**Comment by Viridianus** [comment]
Also, we need a black row that says "Feudal and Clan", like those for Republics and Tribals. It is an important feature that Principalities, Barons and stuff are taken to be feudal.

@Charanraja123: I did not say I would tear off MY arms, those would be arms of the unfortunate formatting offender :D What do you think on the issues I raised - including the suggestion you help in reviewing the changes and comments?

**Comment by Charanraja123** [comment]
> 
> 
> Actually, I thought you or @Urkaun could. I'm kinda dubious to "call to arms" someone who hasn't proven themselves yet, and @Gr770 is clearly doing too much already.
> 
> Should I be that cruel, though? :D India is difficult :D

You see, I will add nothing and only be a nuisance to you. As much as I would love to help you, I have absolutely no fucking knowledge on anything you guys talk about. Most of time I have to google words to understand. 
Now if you want someone to create a new script to write a script for the game. That's something I definitely can do for you xD

**Comment by Charanraja123** [comment]
> Should I be that cruel, though? :D India is difficult :D

Check the forum, we already have a volunteer. Hope he doesn't give up before this week

**Comment by Charanraja123** [comment]
> Btw, what do you think of splitting count-level and duke-level Petty Kings in the master sheet (each defaulting to the corresponding non-independent title)? I finally understood what bothers me that much about the principalities :D

All for this, The defaulting also helps both the people filling up the sheet and the modders.

**Comment by Charanraja123** [comment]
Councillors have adjectives. We need a row for that too I think, otherwise it might awkward?

**Comment by Viridianus** [comment]
1)For English: Shouldn't it go like "$councillor_marshal$'s", with the appropriate localization being inserted from the councillor's "localization"?
2)For other languages this might get trickier. I guess "de" kinda works for French, and I vaguely remember Spanish to have something similar but even German genitive is more demanding (unless we circumvent it with "vom "/"von der " - and even those are gender-sensitive).

**Comment by Charanraja123** [comment]
1) Oh. okay. I thought we had to take proper possessive terms from the cultures language. But I can do that. Shouldn't be a problem
2) Erghh.. I think this is a the start of localization partition. Things gonna become more tough for us xP

**Comment by Viridianus** [comment]
1. No, we shouldn't - by the same token we're not gonna replace "Prince of Novgorod" with "Князь Новгородский" or "Князь Новгорода" but with "Князь of Новгород" - as @Urkaun extensively argued, this would become a language salad out of control.
2. Yes, technically it is - but we can kinda circumvent this by keeping possessives and non-possessives in different files (or, if that's problematic, in different parts of files).

**Comment by Charanraja123** [comment]
1) Makes perfect sense.
2) Not possible for different files, as most localisations we override have to be in that particular file which is as same the vanilla file. Different parts of the same file is okay. I'll try to restrict them to the top of the file If i can. Lets see when we get there

**Comment by Charanraja123** [comment]
I've added some defaults for council titles. Basically if you don't have a specialized female title, it defaults to the male. I hope you guys are okay with that?

**Comment by Viridianus** [comment]
Oh, I've just asked about it in #55 :D I mean, it makes some sense, but that means we'll have to re-code? Maybe this is better solved through the localization files having something like this: `chancellor_councillor_<culture>_female:0 "$chancellor_councillor_<culture>_male$"` not through this oh-so-precious custom localization file?

**Comment by Charanraja123** [comment]
Uhh. I think in both mine and Gib's now deprecated version. We used a similar method of defining male and female only when necessary, so no re-coding is necessary. 
What you suggest is not implementable in Paradox (You can't fill in culture through a script here). Because I'll have to enter it in manually for every culture. Which then means instead of having a single entry for chancellor when there is no female version, I now have two entries which are repetitive. I think the absence of gender in the localization name by itself means the inclusions of all gender in the code for understanding purposes. 
I may have explained it in a little convoluted manner, but I hope you get my point

**Comment by Viridianus** [comment]
> Which then means instead of having a single entry for chancellor when there is no female version, I now have two entries which are repetitive
Yep. Sounds like a good coding practice - so that, if we get a female entry, we can change it without even touching the big "convoluted" code (I don't actually see what's so convoluted about it, but that's a separate issue), only editing the place where the localizations such as "Marszalek" are stored. You'll have to keep the whole set of councillor culture names in _some_ place, right?

**Comment by Charanraja123** [comment]
localization/english/ctp_council_custom_loc_l_english.yml . Is this what you are asking for?

**Comment by Viridianus** [comment]
Well, yes. I suggest that all the coding be uniform (i.e. always presuppose existence of a female version), and the identity be dealt with in this file and this file only.

**Comment by Charanraja123** [comment]
Hmm...Well i dont think there are enough female versions to warrant it to become a default for me. But I could add it,  if you and others want it too. @Gr770 Resolve this for us xD

**Comment by Viridianus** [comment]
@Gr770 - I'll sum it up: we have several requests for you.

1. Rows for independent counts in MT and RC. Please.
2. Technical row saying "Feudal and Clan" in MT, akin to the one in RC or to the "Republic" row below.
3. Decision on the coding problem we discussed immediately above.
4. Decision on "Russian localization" pull request. Pretty please.

**Comment by Charanraja123** [comment]
Interesting . I would have guessed you listing out "Russian Localization" first. xD

**Comment by Gr770** [comment]
I havent pull anything because of the patch today, so that will be done this afternoon

**Comment by Gr770** [comment]
Personally, I dont feel like we need female titles for council positions if none are given. Especially since they will be rare outside of spymaster.

**Comment by Viridianus** [comment]
> Interesting . I would have guessed you listing out "Russian Localization" first. xD

People remember the last thing best ;)

**Comment by Viridianus** [comment]
@Gr770: Regarding rows for Main Titles: I've set the needed structure in Requested Cultures, please do similarly (and, for Force's sake, do it through "Add row", not through dragging) - or just lift the protection if it's too complicated.

**Comment by Viridianus** [comment]
Oh, and also: Central Germanic is somehow under _hard_ lock AND having a very nasty design failure in Chancellor's row. Please set it to warning - or maybe just remove all locks, after a while it does not sound like such a good idea anymore.

**Comment by Gr770** [comment]
Alright, these are done.

**Comment by Charanraja123** [comment]
Since this is "the Issue tab" xD, I thought I would highlight one more problem that I see in the sheet. Obviously this I think is going to a another long discussion, but hopefully it leads to some improvements or clarifications.

## The Curious Case of The Sultan
There are lot of problems with currently modding in the sultan rows. Let me list them out.
1) The fact that we still have not been able to separate them into Malik and Sultan. My method was to change it based on whether they followed a head of faith. Any suggestions that would like to differentiate them is also welcome. Also no rows for Malik in the sheet (But wait, read the post fully)
2) I think this should be its own sheet. Because having sultan flavorisation only does not make sense. What about the rest of the court? Are the english sultants son sill going to be use Prince/Princess flavourisations? In addition this ties heavy into alt history and the reconstruction of Islamic terms into usually historically Christian cultures. I think there is a lot of work to do *especially on the non-coding side), before this can be implemented in a complete manner that adds to the experience of the player. 

What I suggest is 
1) Let sultan be in the sheets (for now) but remove it from the mod so that it doesn't give the feeling of a half worked feature to the player.
2) The second step (In the future, when we have time) would be the migrate it to another sheet and work in a step by step manner to reconstruct or construct those Islamic terms in the context of each Christian cultures language (If this is something you guys want to do). Or just drop the whole thing

**Comment by Viridianus** [comment]
> Alright, these are done.

No. These. Are. NOT. Done. You've hard-locked the whole MT sheet.

**Comment by Charanraja123** [comment]
Cool. I just understood everything. Malik is an regular king unrelated to Islam. I thought every Islam following secular ruler (not submitting to Calpih) all across the world would have Malik. Sorry for the confusion

Also. No more sheets, right? xP

**Comment by Viridianus** [comment]
1. Well, we can specify conditions for Sultans further (say, demand not just islam_religion but some specific faiths or, if possible, tenets), but "Malik" will always be the remainder case, there's _no_ need for three-way King vs. Malik vs. Sultan distinction.
2. Yes, for the following reasons:
2,1. Structure of MT is replicated in RC; any new sheet would have to be likewise replicated, and we _know_ that updating structures is not most people's strongest suit, so that would fall on me.
2.2. Clean-up culture-by-culture will be more complicated if information is dissipated.
2.3. Navigation through the sheets is already suboptimal.
2.4. Search through multiple sheets seems to be unavailable.

**Comment by Charanraja123** [comment]
1. Yes okay. I definitely agree with you on not having a three-way. That's resolved.
2. I agree with that point.
We already have our work cut out for us too. 
If that is everything we needed to discuss, Is this issue going to say open? or can we close it?

**Comment by Viridianus** [comment]
I want it to remain as THE place for me to shout things like: "Who broke the sheet AGAIN?")

**Comment by Charanraja123** [comment]
Noted. I don't think I'll be opening up this Issue tab again.
Good luck @Gr770 xP

**Comment by Viridianus** [comment]
@Charanraja123 - don't hurry out: this is also the place where I can notify you of my progress with EtB modifying and stuff.

**Comment by Gr770** [comment]
BTW the sheet is fully unlocked.

**Comment by Viridianus** [comment]
EtB is renewed and filled till k_lithuania.

**Comment by Viridianus** [comment]
After I found a misplaced comment on Greek Palatine, I finally know why all the "this culture needs to use petty-king system" and "Greek problem is X" comments under the table suddenly disappeared (allowing me to have deleted the now-empty rows). @Gr770 - haven't I specifically mentioned that you should use Add row, not dragging? You simply dragged some titles over the comments, effectively deleting them. It's a good thing I had copied the comments for Volga-Finnic, otherwise we'd lose those as well.

**Comment by Gr770** [comment]
I specifically used add row.

**Comment by Viridianus** [comment]
Then how could it happen? Did someone else come and drag them?

**Comment by Charanraja123** [comment]
I would not rule out that possibility entirely xP. 
Anyway, On the topic of recovering those lost comments, @Gr770 is it possilbe for you to look through the earlier versions and get that comments back. Might be useful now since we added petty kingdoms/kings as a row

**Comment by Charanraja123** [comment]
Also, Emperor Mother does not default to Queen mother right? Just making sure.

**Comment by Gr770** [comment]
1. Comments aren't shown in the sheet history, but I have an idea.

2. No

**Comment by Viridianus** [comment]
1. The point was, they weren't styled as comments (mostly). They were styled as plain text.

2. Shouldn't - but I'm growing more and more dubious about those two rows overall. It's easy to code them, but only Romance, Slavic, Hungarian and Indo-Aryan have something there, and I would not be surprised to find that most of these are literal translations not used by actual cultures (that includes Russian since, as we know, the король-based titles are overall somewhat dubious). Also, there's obvious competition of "Queen/Empress Dowager" logic - or, rather, it should be but no culture shows that in the sheet as of now.

**Comment by Charanraja123** [comment]
1. That is why I was asking, those were text entry and should be in the edit history right?
2. Interesting, If I understand correctly, it actually downgrades our mod quality by being filled with not so accurate terms. In that case may be which just shift it into conditional titles, where the condition is them being of that three culture group?
Also on the topic, Priority for queen mother? What is she is another queen? How does that all work?

**Comment by Viridianus** [comment]
1. Yep.
2. Yes, it may downgrade us. I don't think putting them into conditionals solves the issue - I mean, many of what we have there _now_ . The problem is, we'll have the vanilla's "Queen Mother" and "Empress Mother" all over the place if we don't add our own; maybe having the "Mother" part in plain English but the "Queen/Empress" copied from (consort) queens/empresses could be another compromise. @Gr770, what's your call?
3. If we do implement them, they should have low priority. Like, it's an honorary title of a non-ruler (and usually only saved for those who did not seek another marriage - but I suppose flavorization does not allow for "married =no"). I don't think a duchess (especially a regnant one) which happens to be the emperor's mother could choose to be styled as Empress Dowager rather than Duchess.

**Comment by Charanraja123** [comment]
2. If vanilla, is the problem, we can always override vanilla keys with some impossible conditions and low priority, so that it never gets called, If should not be that hard. Also your suggestion doesn't work on Indian titles. Since its RajaMatha - "King Mother" because it was usually always a male ruler
3. I'll just give it a priority of  around 20, greeter than a countess/ less than a duchess. Or do you want me to go even lower?
Also the problem is it is hard for us to differentiate between reagent and consorts titles, So I have no idea how to approach this

**Comment by Viridianus** [comment]
2. This could also work on vanilla Prince, right? As I mentioned in the forums, we may need them to be off for Germanic. As for Indian - erm, no, it's Mahadevi Mother (sorry for no Macrons) if my idea is implemented.
3. 20 sounds lovely. As for regnant (reagent was funny :D ) and consort... Is it? I thought our only_holder conditions kinda work. The problem is only that, as we code them now, the NAMING convention is "wrong": consort title is the marked one instead of the holder. If _Eorlwif_ will have `duke_feudalclan_female` and _Eorlen_ will have `duke_feudalclan_female_holder` (rather than `_consort` and default), you can always call for the `_holder`-less convention, and it will auto-return the consort if they differ - so have something like Eorlwif Mother...

**Comment by Charanraja123** [comment]
2. It can work on anything, overwriting can be done for any key in the vanilla file which might cause issue with us.  Eh.. Mahadevi is only use for Tamil, I'm very sure the Indian titles use Rajamatha (no Macrons/Diacritics/Accents idk which word to use xP)

3. Yeah, four years of chemical engineering does that xP. Also you are right, I should have gone for `_holder` instead of `_consort` makes much more sense while coding.
Check above point about India

That just feels wrong. I'd rather not have it lol. The tool tips already say your mother, So seems unnecessary.

**Comment by Viridianus** [comment]
_Interlude_: macron is the specific diacritic of a line above the letter, usually used for long vowels. Diacritic is a general term for all those signs you can append to a letter. Accent is either a phonological entity or a subset of diacritics traditionally associated with it (acute, grave and, for historical reasons, French accent circonflex like î; sometimes also weird tonal accents). _End interlude._

2. I see what's in there now, but are you sure that's what we should have?
3. Please make a fix for that, yeah.
It's not only about your _own_ mother, though :D There will be many kings and many mothers thereof.

**Comment by Charanraja123** [comment]
Interlude helped, Thanks.
I have a curious question though, may be a little off topic, is this word "accent" the inspiration of the other "Accent" (People have while talking based on their geographical region) or is it vice versa.

2. I mean, I made a big post about how everything is India doesn't make any sense.
3. Sure, after I finish Frankish

**Comment by Viridianus** [comment]
Vice versa - people with accent were thought to make strange accents ;)

2. Yet rajamatha suddenly makes sense?)))

You ask, it's your pain)

**Comment by Charanraja123** [comment]
2. Someone Put it there for a reason , Maybe I just don't know enough

**Comment by palfourti** [comment]
Rajamata is _slightly_ different because it actually means king's mother, not queen mother :)

That said I only put it there because it seemed to have been an actual historical title rather than just a translation (then again so probably is every language's translation of queen mother), but I don't think its absence will be particularly missed. This was before I learned we weren't bothering with just straight translations of QM/consort for those titles as well.

**Comment by Viridianus** [comment]
If it's an actual historical title it deserves to remain, so the Indian situation is solved. Yet I actually think that many _other_ entries in the row are just translations some overeager guy/gal made in order not to leave anything behind.

**Comment by Charanraja123** [comment]
I can confirm that it is an Historical Title, and a quite popular one at that too,
The doubt I had/have all across India, is whether every culture used the same thing. But that is a discussion for another place.

**Comment by Charanraja123** [comment]
I mean if you want, we could not code in Queen mother for the time being. Or we could just put them in and then correct them later.

**Comment by palfourti** [comment]
At the very least the Romance ones are probably the most suspect since they seem to be the ones that also include literal translations for consorts (or DID include, it seems some of them have been cleaned up).

**Comment by Charanraja123** [comment]
@palfourti are you the one reconstructing those Indian Culture titles? If you are, how confident are in those titles. I'm asking because I do not want to give people wrong/meaningless terms, which in a certain point of view might be worse than using a modern term.
If @palfourti is not the guy working on them. @Viridianus , Can you verify them, I just want to make sure they are not bad reconstructions.

**Comment by palfourti** [comment]
Yes, I'm the one doing those, hopefully I explained myself well enough on the comments.

**Comment by Gr770** [comment]
For Norse Culture week, I am going to leave the norse pagan area unhidden. @Viridianus hopefully shouldn't have to fix formatting errors from people rehiding the section

**Comment by Viridianus** [comment]
I sure hope so ;) Leaving our major Norse problems in the forum.

**Comment by Viridianus** [comment]
Empires of Russia and Khazaria (and everything before them) are crosschecked in EtB, as well as kingdoms of Mordvinia and Volga Bulgaria.

**Comment by Charanraja123** [comment]
Moved to Discord
